ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ranger/RangerRestClientImpl.java

```diff
@@ -428,10 +428,7 @@ public RangerExportPolicyList readRangerPoliciesFromJsonFile(Path filePath,
                                                                HiveConf conf) throws SemanticException {
     RangerExportPolicyList rangerExportPolicyList = null;
     Gson gsonBuilder = new GsonBuilder().setDateFormat("yyyyMMdd-HH:mm:ss.SSS-Z").setPrettyPrinting().create();
-    try {
-      FileSystem fs = filePath.getFileSystem(conf);
-      InputStream inputStream = fs.open(filePath);
-      Reader reader = new InputStreamReader(inputStream, Charset.forName("UTF-8"));
+    try (Reader reader = new InputStreamReader(filePath.getFileSystem(conf).open(filePath), Charset.forName("UTF-8"))) {
```

Comment:
still leaks InputStream - in try should be line for InputStream and Reader

Reply:
@slachiewicz Thanks for your review, I've updated the patch